test/higherorderfns.jl

```diff
@@ -709,24 +709,43 @@ end
     @test extrema(f, x) == extrema(f, y)
     @test extrema(spzeros(n, n)) == (0.0, 0.0)
     @test extrema(spzeros(n)) == (0.0, 0.0)
-    @test_throws ArgumentError extrema(spzeros(0, 0))
-    @test_throws ArgumentError extrema(spzeros(0))
+    # @test_throws "reducing over an empty" extrema(spzeros(0, 0))
+    # @test_throws "reducing over an empty" extrema(spzeros(0))
```

Comment:
Does this work after https://github.com/JuliaLang/julia/pull/43604?

```suggestion
    # TODO: enable the following once https://github.com/JuliaLang/julia/pull/43604 is merged
    @test_broken "reducing over an empty" extrema(spzeros(0, 0))
    @test_broken "reducing over an empty" extrema(spzeros(0))
```